packages/dataviews/src/components/dataviews-filters/filter-summary.tsx

```diff
@@ -229,19 +232,33 @@ function OperatorSelector( {
 export default function FilterSummary( {
```

Comment:
I feel the name of this component is a bit weird, this seems to be the actual filter (not a summary)

Reply:
Indeed, it can be confusing. This is the code where the component is used. Renaming it to 'Filter' would be clearer. I'll rename it.

https://github.com/Automattic/wp-calypso/blob/e29a70a654a8d75e388860e171a73d2026b3c701/packages/dataviews/src/components/dataviews-filters/index.tsx#L195-L208